modules/ocf/manifests/firewall/output_printers.pp

```diff
@@ -1,11 +1,9 @@
 class ocf::firewall::output_printers {
   # Allow output to printers
-  ['papercut', 'pagefault'].each |$printer| {
-    firewall { "899 allow output to ${printer}":
-      chain       => 'PUPPET-OUTPUT',
-      destination => $printer,
-      proto       => 'all',
-      action      => 'accept',
-    }
+  firewall_multi { '899 allow output (printers)':
+    chain       => 'PUPPET-OUTPUT',
+    destination => ['papercut', 'pagefault'],
```

Comment:
hardcoding hostnames seems like it could be risky for future maintenance

Reply:
What should we do instead?

Reply:
@abizer